src/components/Footer.tsx

```diff
@@ -306,132 +298,69 @@ const Footer = ({ lastDeployLocaleTimestamp }: FooterProps) => {
     },
   ]
 
-  const hoverStyles = {
-    textDecor: "none",
-    color: "primary.base",
-    _after: {
-      color: "primary.base",
-    },
-    "& svg": {
-      fill: "primary.base",
-    },
-  }
-
-  const linkProps = {
-    isPartiallyActive: false,
-    textDecor: "none",
-    color: "body.medium",
-    fontWeight: "normal",
-    _hover: hoverStyles,
-    sx: {
-      "& svg": {
-        fill: "body.medium",
-      },
-    },
-  }
-
   return (
-    <Box as="footer" py="4" px="8">
-      <Flex
-        justify={{ base: "center", md: "space-between" }}
-        alignItems="center"
-        flexWrap="wrap"
-        gap={8}
-        pt={4}
-        pb={4}
-        borderTop={"1px solid"}
-        borderColor={"body.light"}
-      >
-        <Text fontSize={"sm"} fontStyle={"italic"} color={"body.medium"}>
+    <footer className="px-8 py-4">
+      <div className="flex flex-wrap items-center justify-center gap-8 border-t border-body-light pb-4 pt-4 md:justify-between">
+        <p className="text-sm italic text-body-medium">
           <Translation id="website-last-updated" />: {lastDeployLocaleTimestamp}
-        </Text>
+        </p>
 
-        <Button
-          leftIcon={<IoChevronUpSharp />}
-          variant="outline"
-          isSecondary
-          onClick={() => scrollIntoView("__next")}
-        >
-          {t("go-to-top")}
+        <Button variant="secondary" onClick={() => scrollIntoView("__next")}>
+          <ButtonIcon icon={<IoChevronUpSharp />} /> Go to top
         </Button>
-      </Flex>
+      </div>
 
-      <SimpleGrid
-        gap={4}
-        justifyContent="space-between"
-        templateColumns={{
-          base: "auto",
-          sm: "repeat(2, auto)",
-          md: "repeat(3, auto)",
-          xl: "repeat(6, auto)",
-        }}
-      >
+      <div className="grid auto-cols-auto justify-between gap-4 sm:grid-cols-2 md:grid-cols-3 xl:grid-cols-5">
         {linkSections.map((section: FooterLinkSection, idx) => (
-          <Box key={idx}>
-            <Heading as="h3" fontSize="sm" lineHeight="base" my="1.14em">
+          <div key={idx}>
+            <h3 className="my-5 text-sm font-bold">
               <Translation id={section.title} />
-            </Heading>
-            <List fontSize="sm" lineHeight="base" fontWeight="normal" m="0">
+            </h3>
+            <List className="m-0 mb-4 list-none text-sm">
```

Comment:
`mb-4`?

<img width="851" alt="image" src="https://github.com/user-attachments/assets/f7b5678c-ed21-42eb-aba7-fd0ff9442d76">

> _Production_

I don't see this in production; are we desiring this change?

Reply:
We currently have an mb in the `li`, on the ListItems, in production. I replaced it with this mb placed on the parent `ul'. Basically, it is the space between the list section and the next component below it. I moved the place where we declare this space since I think it is more appropriate to place it on the parent than on the children.